Bookcrossing.Host/Controllers/PublisherController.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Bookcrossing.Application.Commands.Publisher;
+using Bookcrossing.Application.Queries.Publisher;
+using Bookcrossing.Contracts.Abstractions.RequestFeatures;
+using Bookcrossing.Contracts.DataTransferObjects.Creation;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Threading.Tasks;
+
+namespace Bookcrossing.Host.Controllers
+{
+    [Route("api/publisher")]
+    [ApiController]
+    public class PublisherController : ControllerBase
+    {
+        private readonly IMediator _mediator;
+
+        public PublisherController(IMediator mediatr)
+        {
+            _mediator = mediatr;
+        }
+
+        [HttpGet("{id}")]
+        public async Task<IActionResult> GetPublishers(Guid id)
+        {
+            var result = await _mediator.Send(new GetPublisherById(id));
+
+            return Ok(result);
+        }
+
+        [HttpGet]
+        public async Task<IActionResult> GetWithParams([FromQuery] AuthorPublisherParams parameters)
```

Comment:
maybe better to add something like 'search' in the route. Just because as for me it seems more obvious to get all models using current route. 
So could be api/publisher/search